.github/workflows/terraform-observe_scheduler.yaml

```diff
@@ -57,7 +57,32 @@ jobs:
           OBSERVE_DOMAIN: ${{ secrets.TERRAFORM_MODULES_TEST_OBSERVE_DOMAIN }}
           OBSERVE_USER_EMAIL: ${{ secrets.TERRAFORM_MODULES_TEST_OBSERVE_USER_EMAIL }}
           OBSERVE_USER_PASSWORD: ${{ secrets.TERRAFORM_MODULES_TEST_OBSERVE_USER_PASSWORD }}
-          
+
           GITHUB_WORKSPACE: ${{ github.workspace }}
           GITHUB_SHA: ${{ github.sha }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  # Validate App Icon URL is internal
+  validateAppIcon:
```

Comment:
Job title should match the case of others. Avoid camel case in actions, dash (kebab) case is the default, snake is also commonly used.